README.md

```diff
@@ -17,7 +17,33 @@ If this proposal stands, we plan to extend the [Background Sync](https://wicg.gi
 specification to describe this ability as well, and move the document to the standardization track.
 
 ## Use cases
-  * todo
+  Many apps benefit from updating their state and content when the device has
+  network connectivity, before the user navigates to their web page.
+  This skips the otherwise slow and laggy experience that the
+  user would otherwise go through if the app was being updated while being used.
```

Comment:
While I agree with the premise, you're effectively describing "loading a page" here. What I think would make for a stronger argument is a website that is offline-enabled, provides near-instant loading, but potentially shows outdated content or has to swap out content after a wee while when loading it from the network.

Reply:
Done.